docs/en/pid-eaa-issuance.rst

```diff
@@ -439,50 +454,152 @@ If the checks defined above are successful the Wallet Instance proceeds with the
+
+
+Re-Issuance Flow
+----------------
+
+Re-issuance refers to the replacement of Digital Credentials already stored in a Wallet Instance with new Digital Credentials of the same document type. The new Digital Credentials MUST be issued by the same Credential Issuers that originally issued the existing ones to the same Wallet Instance storing them. 
+
+To facilitate this, particularly in scenarios where User authentication is not strictly required, a Refresh Token (RT) flow MAY be used (see Section :ref:`Refresh Token Flow <Refresh Token Flow>` for more details). An Access Token obtained as a result of a Refresh Token flow  MUST NOT be used to issue a Digital Credential that is not present in the Wallet Instance (first-time-issuance). The Refresh Token mechanism enables automated Credential replacement, streamlining the process for both the Credential Issuer and the User.
+
+The re-issuance process outlined in this section is limited to the following scenarios:
+
+  - Data model/format technical update;
+  - User's attribute set update.
+
+In the first case, the new Digital Credential's Users attribute set will match the original one. For example, a Credential Issuer may need to update the Digital Credential metadata or data format without changing the User's attribute set. In this case, the direct involvement of the User is not mandatory for the replacement and storage of a Digital Credential. 
+
+In the second case, Credential Issuers may also need to modify one or more User's attribute values during re-issuance. In this case, the Wallet Instance MUST inform the User that the attribute data set has been changed and MUST then request the User's authorization to store the new Digital Credential. 
+
+In both cases, the newly issued Digital Credential MUST have the same expiry date as the previous one. 
+
+Re-issuance after Digital Credential expiration MUST always require User authentication.
```

Comment:
@fmarino-ipzs there are points for IA WG

Reply:
This is an important requirement for us.  If we allow indefinite Credential renewals without proper User authentication, any security problem that occurred during the first-issuance would propagate indefinitely. This is one of the reasons why documents expire in the physical world and new identity proofing is required to issue a new document. Probably, we could relax this requirement in case a Digital Credential doesn't require User authentication during the first-issuance. 

@giadas ^^